test/functional/wallet_migration.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test Migrating a wallet from legacy to descriptor."""
+
+import os
+
+from test_framework.key import ECKey
```

Comment:
The linter is complaining this going unused.

Reply:
Fixed.